frontend/app/auth/login.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+  Image,
+} from "react-native";
+import { useRouter } from "expo-router";
+
+export default function Login() {
+  const router = useRouter();
+
+  return (
+    <View style={styles.wrapper}>
+      <View style={styles.container}>
+        <Text style={styles.title}>Login Page</Text>
+
+        <TextInput
+          style={styles.input}
+          placeholder="Email or Phone"
+          placeholderTextColor="#000"
```

Comment:
gawa ka Theme.ts

sa Themes.ts. palitan mo din values.
```
const Colors = {
  primary: "#1E90FF",
  secondary: "#FFD700",
  background: "#FFFFFF",
  text: "#333333",
  error: "#FF4D4D",
}

export const Theme = {
  background: Colors.background,
  text: Colors.text,
  primary: "#007AFF",
};
```


```suggestion
import { Theme } from 'Theme'
          placeholderTextColor={theme.text}
```

Reply:
Done